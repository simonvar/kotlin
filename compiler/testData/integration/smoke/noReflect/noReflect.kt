package noReflect

fun main() {
    String::class.annotations
}