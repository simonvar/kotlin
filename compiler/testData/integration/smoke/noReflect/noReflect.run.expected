ERR:
Exception in thread "main" kotlin.jvm.KotlinReflectionNotSupportedError: Kotlin reflection implementation is not found at runtime. Make sure you have kotlin-reflect.jar in the classpath
	at kotlin.jvm.internal.ClassReference.error(ClassReference.kt:84)
	at kotlin.jvm.internal.ClassReference.getAnnotations(ClassReference.kt:27)
	at noReflect.NoReflectKt.main(noReflect.kt:4)
	at noReflect.NoReflectKt.main(noReflect.kt)

Return code: 1
